alfresco-transform-imagemagick/alfresco-transform-imagemagick/src/main/resources/engine_config.json

```diff
@@ -0,0 +1,1158 @@
+{
```

Comment:
I wonder why this appears as a diff in the PR rather than a rename ?

Reply:
It's because the file location has changed slightly. Previously it was in `.../src/resources/` it is now in `.../src/main/resources/`